app-ios/Sources/SessionFeature/SessionTagsView.swift

```diff
@@ -20,8 +59,31 @@ extension TimetableItem {
             Tag(text: $0, color: AssetColors.secondaryContainer.swiftUIColor)
         }
     }
+
     private var durationInMinutes: Int64 {
         let durationInSeconds = self.endsAt.epochSeconds - self.startsAt.epochSeconds
         return durationInSeconds/60
     }
 }
+
+private struct HeightPreferenceKey: PreferenceKey {
+    static func reduce(value: inout CGFloat, nextValue: () -> CGFloat) {}
+    static var defaultValue: CGFloat = 0
+}
+
+private struct HeightReaderView: View {
+    @Binding var height: CGFloat
+    
+    var body: some View {
+        GeometryReader { geometry in
+            Color.clear
+                .preference(
+                    key: HeightPreferenceKey.self,
+                    value: geometry.frame(in: .local).size.height
+                )
+        }
+        .onPreferenceChange(HeightPreferenceKey.self) { height in
+            self.height = height
+        }
+    }
```

Comment:
https://github.com/DroidKaigi/conference-app-2022/pull/403/commits/e50c77f6cbcde2bedaea1a7fda26728b9c7bc433#r970761357
> How about use [LazyHGrid](https://developer.apple.com/documentation/swiftui/lazyhgrid) with adaptive column?

I couldn't make it work properly with LazyGrid so instead tried determining the height of the TagsView and setting .frame(height:) to its parent Vstack.

Reply:
Okey!
I'll try to implement with `LazyGrid` in other issues and PR! This is ok for now 👍🏼